packages/react-dom-bindings/src/server/fizz-instruction-set/ReactDOMFizzInlineCompleteSegment.js

```diff
@@ -0,0 +1,5 @@
+import {completeSegment} from './ReactDOMFizzInstructionSet';
+
+// This is a string so Closure's advanced compilation mode doesn't mangle it.
```

Comment:
nit: I don’t believe we’re using the “advanced” GCC mode.

Reply:
We are in the script I just added 😆